docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    CS1 = plt.contour(
</pre><p>maybe use a for loop? it also seems _ = plt.contour( .. ) could be used since you are not using CS1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>